README.md

```diff
@@ -1,9 +1,11 @@
 # cfgov-refresh
 
-Layout and content for the consumerfinance.gov Refresh project.
+This repository contains the redesign-in-progress of consumerfinance.gov. This includes front-end assets and build tools, and configuration for [Sheer](https://github.com/cfpb/sheer) to load content from Wordpress and Django back-ends to elasticsearch to render the site.
 
 ### This project is a work in progress
 
+![screen shot 2015-01-28 at 10 47 12 am](https://cloud.githubusercontent.com/assets/235397/5940816/3529246a-a6db-11e4-9bfc-76d5b5220a36.png)
```

Comment:
There's no need to use an absolute link. Just commit the image file as `screenshot.png` and link to it in the markdown like this `[screenshot](screenshot.png)`.